gaphor/SysML/toolbox.py

```diff
@@ -31,6 +25,31 @@ def metaclass_config(new_item):
     new_item.subject.name = "Class"
 
 
+def create_association(
+    assoc_item: uml_items.AssociationItem, part_association: bool
+) -> None:
+    assoc = assoc_item.subject
+    assoc.memberEnd.append(assoc_item.model.create(UML.Property))
```

Comment:
Just an assignment will work here as well. It’s a feature of the model.

Reply:
I was originally trying to do:
```
assoc.memberEnd = assoc_item.model.create(UML.Property)
assoc.memberEnd = assoc_item.model.create(UML.Property)
```
Unfortunately that causes mypy to detect that as an error:
```
gaphor/SysML/toolbox.py:35: error: Value of type "Property" is not indexable  [index]
gaphor/SysML/toolbox.py:36: error: Value of type "Property" is not indexable  [index]
```

Reply:
Ah, then using `append()` is the better way. It's more explicit anyhow.